### ISSUE TITLE
React Native Packager doesn't scroll

### ISSUE THREAD

**Issue by mikelambert** [facebook/nuclide#464]
### Issue and Steps to Reproduce

I start the Nuclide React Native Packager. Everything works fine, but it never scrolls. If I leave it running for awhile, this makes it really hard to find/see any errors, since:
- It's not scrolled to the bottom
- hitting "End"/"Page Down" don't actually work
- there are no scrollbars I can use to quickly jump to the bottom
- I have to scroll manually with my mouse/trackpad all the way to the bottom

### Expected Behavior
Tell us what should happen.
- Stay scrolled to the bottom as new input comes in
- Support "End" and "Page Down" (and related navigation keys)
- Maybe scrollbars, if it's not too difficult?

### Versions
* Atom: 1.7.2
* Nuclide: 0.130.0
* Client OS: Mac OS X 10.11.4
* Server OS (optional):

### Additional Details
* Installed packages (`apm ls --installed`):
Community Packages (13) /Users/lambert/.atom/packages
├── autocomplete-python@1.7.2
├── haskell-grammar@0.4.0
├── highlight-selected@0.11.2
├── language-babel@2.17.3
├── language-ini@1.16.0
├── language-ocaml@1.1.2
├── language-thrift@1.0.2
├── linter@1.11.4
├── minimap@4.22.1
├── nuclide@0.130.0
├── python-tools@0.6.7
├── sort-lines@0.14.0
└── tool-bar@0.4.0

└── (empty)

**Comment by johnislarry** [comment]
What do you think @matthewwithanm

**Comment by matthewwithanm** [comment]
Thanks for the issue, @mikelambert! We're working on updating how the output is displayed so I'm not sure how much time we want to spend on updating the old version. Still, I can at least make this autoscroll when you're at the bottom in the meantime!

This is just an Atom Text editor, so Page Up/Down should work as normal. Did the pane have focus when you tried?

**Comment by mikelambert** [comment]
Clicking on the React Native Server tab is not sufficient to give it focus (which is what I had tried at first), clicking to give focus will let Page Up/Down work. (Maybe it's worth auto-focusing, in your new version?)

But from your comment not mentioning Home/End, I found https://github.com/atom/atom/issues/7349, namely Atom prefers to support the Windows convention for Home/End keys, even on the Mac platform. No idea why, but I guess at least I can override it to work for me. Thanks.
